tests/routes/stats.spec.js

```diff
@@ -0,0 +1,49 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import sinon from 'sinon';
+import { app, db } from '../../server';
+import { createUser, createArticle } from '../helpers';
+import { transporter } from '../../utils/mailer';
+
+const { expect } = chai;
+
+let mockTransporter;
+let profile;
+let article;
+
+let newArticle = {};
+
+chai.use(chaiHttp);
+
+describe('USER STATS', () => {
+  before(async () => {
+    mockTransporter = sinon.stub(transporter, 'sendMail').resolves({});
+    profile = {
+      firstName: 'vincent',
+      lastName: 'hamza',
+      username: 'wkev',
+      password: '12345678',
+      email: 'pro@gmail.com'
+    };
+    article = {
+      title: 'React course by hamza',
+      description: 'very good book',
+      body: 'learning react is good for your career...'
+    };
+  });
+  after(async () => {
+    mockTransporter.restore();
+    await db.User.destroy({ truncate: true, cascade: true });
+  });
+
+  it('Increment Article view after reading time elapses', async () => {
+    const newUser = await createUser(profile);
+    newArticle = await createArticle({ ...article, authorId: newUser.id });
+    const { slug } = newArticle;
+    const res = await chai
+      .request(app)
+      .patch(`/api/v1/stats/${slug}`).send();
```

Comment:
It will be better to check for the status code of the response gotten because if the test fails the status code could lead to finding where the problem is from

Reply:
done